### ISSUE TITLE
[REQUEST] Strygossi Renaissance 

### ISSUE THREAD

**Issue by RexofRex-GitHub** [Hexenstag-Team/WH-Hexenstag-Dev#42]
Use the Roman Renaissance mechanic for the Empire of Strygos to spread a "Strygossi" culture which replaces the Strygan culture.

Since the Strigany are a splintered and heavily mixed people they obviously lost much of their Roots and i think its reasonable to assume that they would try to look at their past and try to revive their culture after they succesfully take back their ancient Homeland.  Also lets the Stirgoi Vampires start with the Strigany Culture since they usually rule over Strigany anyways who struggle to survive and cant really assemble stuff like their Blood Legionaries or build their unique structures. 

TL:DR Remove Strygan Culture from game start, use Roman Renaissance to get access to it instead, give all Strigoi Strygany culture aswell to represent that they still rule over Strigany Mortals and prevent the building of the unique buildings/retinues for that culture until their Empire is restored.
